src/solvers/CMakeLists.txt

```diff
@@ -51,6 +55,7 @@ list(REMOVE_ITEM sources
     ${lingeling_source}
     ${booleforce_source}
     ${minibdd_source}
+    # ${ipasir_source}
```

Comment:
Commented out option?

Reply:
No - this is a weird mechanism in the cmake file, where the files specified there are _removed_ from the build, and then re-introduced later, when the particular solver to be linked against has been identified.

I haven't understood the mechanism exactly (how it reintroduces the files for, say, minisat or glucose) so I've left this there as a reminder for me to investigate further. If I remove the commenting out of the ipasir sources there, what happens is that we get a linking error when building `smt2_solver`, where it complains that it can't find references for `satcheck_ipasir` (i.e. failing to build `satcheck_ipasir.cpp`).